src/modules/user-sessions/login.js

```diff
@@ -0,0 +1,105 @@
+import { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link, useNavigate } from 'react-router-dom';
+import { userSession } from '../../redux/user/session-redux';
+
+const Login = () => {
```

Comment:
- [ ] I can signup successfully, do reservations, and add and delete users.  However, when I logout, I can't login again, even though the rails console indicates that the user exists in the database.  

If this is a bug, kindly fix it so your app works as intended. 👍 

This was tested both on Firefox and Chrome.